python/grass/docs/conf.py

```diff
@@ -73,15 +73,16 @@
     ),
 )
 
-from grass.script import core
+if True:
+    from grass.script import core
```

Comment:
What is the condition for?

Reply:
If I understand this correctly `# noqa: E402` would be more appropriate here, no?

Reply:
Added it avoids E402 (module level import not at top of file) issues. The other options are 1) # noqa: E402 to all imports not at top level, 2) #flake8: noqa at top of Python file, or 3) adding the files in .flake8 "per-file-ignores". Let me know your preference.

Reply:
Option 1 would be most appropriate here.